bookkeeper-server/src/test/java/org/apache/bookkeeper/client/api/BookKeeperBuildersOpenLedgerTest.java

```diff
@@ -58,20 +56,20 @@ public class BookKeeperBuildersOpenLedgerTest extends MockBookKeeperTestCase {
 
     private boolean withRecovery;
 
-    public BookKeeperBuildersOpenLedgerTest(boolean withRecovery) {
+    public void initBookKeeperBuildersOpenLedgerTest(boolean withRecovery) {
         this.withRecovery = withRecovery;
     }
 
-    @Parameterized.Parameters(name = "withRecovery:({0})")
     public static Collection<Object[]> data() {
         return Arrays.asList(new Object[][]{
                 {true},
                 {false}
         });
     }
 
-    @Test
-    public void testOpenLedger() throws Exception {
+    @MethodSource("data")
```

Comment:
In junit5, it's better to use `@ValueSource(booleans =  {true, false})` for cases where there's a single boolean parameter.

Reply:
@lhotari Thanks，It's a better choice.